components/alert/alert-toast.js

```diff
@@ -21,6 +21,9 @@ const TOAST_SPACING_SMALL = 0.3;
 
 const mediaQueryList = window.matchMedia('(max-width: 615px)');
 
+let ALERT_HAS_FOCUS = false; // if this alert or sibling alert is focused on
+let ALERT_HAS_HOVER = false; // if this alert or sibling alert is hovered on
```

Comment:
An example usecase (added a visual diff for this) is:
- toast 1 has focus
- toast 2 is hovered on then hover is remove. 

Without being aware of the state of toast 1 (i.e., that some toast is either being focused or hovered on), toast 2 would just close.

The code changes below do assume that only one toast can be hovered or focused on at once.

Reply:
At first I was confused as to how globals could work for this, but they do since at most one can be hovered and/or focused at once -- cool!